aries-site/src/pages/index.js

```diff
@@ -1,69 +1,100 @@
+  Quote,
+  WhatIs,
+} from '../components/home';
+import { Layout /* , PageIntro */ } from '../layouts';
+import { getPageDetails } from '../utils';
 
 const title = 'Home';
 const pageDetails = getPageDetails(title);
-const cards = getCards();
-const featured = [
-  'Designer Guidance',
-  'Forms',
-  'Header',
-  'Human Centered',
-  'Developer Guidance',
-  'Button',
-  'Navigation',
-  'Icons',
-  'Search',
-  'Typography',
-];
 
-const featuredCards = [];
-const cardOrder = [];
+const calcPad = size => {
+  const val = size !== 'small' ? 'xlarge' : 'large';
+  return val;
+};
 
-cards.forEach(card => {
-  if (featured.includes(card.name)) featuredCards.push(card);
-  else cardOrder.push(card);
-});
+const Intro = ({ children }) => {
+  const size = useContext(ResponsiveContext);
+  return (
+    <Box height={{ min: 'medium' }} justify="center">
+      <Grid gap="large" columns={size === 'small' ? ['auto'] : ['2/3', 'auto']}>
+        <Card background="none" elevation="none">
+          {children}
+        </Card>
+        {size !== 'small' && (
+          <Card background="none" elevation="none" height="small" />
+        )}
+      </Grid>
+    </Box>
+  );
+};
 
-featuredCards.sort(
-  (a, b) => featured.indexOf(a.name) - featured.indexOf(b.name),
-);
+Intro.propTypes = {
+  children: PropTypes.node,
+};
 
-cardOrder.splice(0, 0, ...featuredCards);
+const Index = () => {
+  const size = useContext(ResponsiveContext);
+  const backgroundImage = {
+    src: { dark: '/home.svg', light: '/home.svg' },
+    alt: 'HPE Design System',
+    margin: { top: '75px', left: '0px' },
+    style: {},
+    position: size !== 'small' ? 'right' : 'left',
+  };
 
-const Index = () => (
-  <Layout
-    backgroundImage={{
-      src: { dark: '/cards-dark.svg', light: '/cards.svg' },
-      alt: 'HPE Design System',
-      margin: { top: '50px', left: '-150px' },
-      style: { transform: 'scale(1.4)', transformOrigin: 'top left' },
-      small: {
-        margin: { left: '-75px', top: '-75px' },
-      },
-      useGrid: true,
-    }}
-    title={title}
-    isLanding
-  >
-    <Meta title={title} description={pageDetails.seoDescription} />
-    <Box gap="large">
-      <PageIntro>
-        <Box justify="center" fill>
-          <Heading margin="none">HPE Design System</Heading>
-          <Paragraph size="large" fill>
-            The Design System was created to empower designers, developers, and
-            others to contribute to making great experiences for the customer.
-          </Paragraph>
+  return (
+    <Layout backgroundImage={backgroundImage} title={title} isLanding pad={{}}>
+      <Meta title={title} description={pageDetails.seoDescription} />
+      <Box>
+        <Intro position="left">
+          <Box pad={calcPad(size)}>
+            {size === 'small' && (
+              <Card background="none" elevation="none" height="small" />
+            )}
+            <Heading margin="none" size="large">
+              Design, develop and deliver
```

Comment:
Could we break "and deliver" so it's always on the second line at full width

Reply:
Done